masterlist.yaml

```diff
@@ -14921,7 +14929,7 @@ plugins:
         subs: [ 'The Great City of Solitude' ]
         condition: 'active("The Great City of Solitude.esp") and not active("CC_TGCS_Patch.esp")'
     clean:
-      - crc: 0xA69C30B9
+      - crc: 0x1C7037D9
```

Comment:
This is the checksum of `Convenient Carriages.esp` with the **"Beyond Skyrim - Bruma SE" and "Helgen Reborn"** & **Travel speed tweak - Requires SKSE** options selected in the FOMOD installer. The previous checksum was with the **No optional mods installed** & **Travel speed tweak - Requires SKSE** options selected. It seems excessive to include cleaning data for every possible combination, so I think it'd be best to just report it for the default version & maybe the default + SKSE combo.